File: app/src/main/java/com/devdream/cookall/core/services/login/LoginService.java
package com.devdream.cookall.core.services.login;

import com.devdream.cookall.core.dto.UserAuthDTO;
import com.devdream.cookall.core.listeners.OnLoginFetchedListener;

public interface LoginService {

    void login(UserAuthDTO userDTO, final OnLoginFetchedListener onLoginFinishedListener);

}

File: app/src/main/java/com/devdream/cookall/core/services/login/LoginService.kt
package com.devdream.cookall.core.services.login

import com.devdream.cookall.core.dto.UserAuthDTO
import com.devdream.cookall.core.listeners.OnLoginFetchedListener

interface LoginService {

    fun login(userDTO: UserAuthDTO, onLoginFinishedListener: OnLoginFetchedListener)

}
